packages/cascara/src/modules/DataJson/DataJson.js

```diff
@@ -31,37 +33,39 @@ const DataJson = ({
   const { isEditing, formMethods } = useContext(ModuleContext);
   const jsonValue = JSON.stringify(value, null, 4);
 
+  const conditionalLabelProps = getConditionalLabelProps(label, isLabeled);
+
   const renderEditing = (
-    <label htmlFor={label}>
-      {label && isLabeled && <span className={styles.Label}>{label}</span>}
-      <Input
-        {...rest}
-        as={TextareaAutosize}
-        className={styles.Input}
-        defaultValue={jsonValue}
-        id={label}
-        name={attribute || label}
-        ref={formMethods?.register}
-      />
-    </label>
```

Comment:
@lordpixel if we look at the code before and after, and we look at the number of DOM nodes that this code renders before and after, and their parent/child relationships, we are still changing that here.

It is extremely important that we are still rendering the exact same number of nodes with the same structure in both display and editing mode, and we need to be maintaining those with these changes.